src/base/utils/foreignapps.cpp

```diff
@@ -247,12 +259,9 @@ PythonInfo Utils::ForeignApps::pythonInfo()
 {
     static PythonInfo pyInfo;
     if (!pyInfo.isValid()) {
-#if defined(Q_OS_UNIX)
-        // On Unix-Like systems python3 should always exist
-        // https://www.python.org/dev/peps/pep-0394/
         if (testPythonInstallation("python3", pyInfo))
             return pyInfo;
-#endif
+
         // Look for "python" in Windows and in UNIX if "python3" is
```

Comment:
It seems that the mention of any operating systems in this comment is now meaningless.